### ISSUE TITLE
[Idea] Verification Bounties

### ISSUE THREAD

**Issue by tsitu** [DevJackSmith/mh-helper-extension#38]
Re: #29 (@tehhowch)

It does seem reasonable to assume a linear relationship between total rage and AR for MBW. You guess at a starting point of 50% AR at 75 rage - would it be worth it to set up a bounty for folks to verify that estimate with LL & Taunting at 25/25/25? And possibly also at 100 and 125 total rage?

More broadly, would it be worth it to set up bounties for other areas of analytic interest? For the test suite, pre/post hunt user objects during stage transitions might be easier to gather via incentivized crowdsourcing.

**Comment by DevJackSmith** [comment]
i agree, we just need to implement that giveaways idea we had, so it's automated. Or i can put my sb+ on the line if someone wants to do it manually.
